server/entities/classifier.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC
+
+
+class Classifier(ABC):
+    name = "sampleName"
+    sample_data = None
```

Comment:
These are not associated with any specific Classifier. Think of these as being the static variables on a java class.

Reply:
Even the sample_data?

Reply:
Yes, they should be defined in __init__(self)